### ISSUE TITLE
Input not working on Android / pixi v8

### ISSUE THREAD

**Issue by Makio64** [pixijs/ui#170]
### Expected Behavior
When I type a text it appears correctly

### Current Behavior
I can type text but the text isnt update in pixi. It works well on computer ( windows/mac ) as well as on iOS 

### Steps to Reproduce
Open the storybook on Android : https://pixijs.io/ui/storybook/?path=/story/components-input-use-graphics--use-graphics

### Environment

- **version**: `pixi.js 8.1.6` / `@pixi/ui 2.1.1`
- **Browser & Version**: Chrome Android latest
- **OS & Version**: _e.g. Android ( all , samsung, pixel, rednote, etc.. )
- **Running Example**: [_url_](https://pixijs.io/ui/storybook/?path=/story/components-input-use-graphics--use-graphics)

**Comment by Makio64** [comment]
Current workaround working on android :
 
```
                // init code 
                // this.input = new Input(... 

                // add a requestAnimationFrame :  
		stage.onUpdate.add(this.update)
	}

	update = ()=>{
		if(this.input.input){
			console.log(this.input.input.value)
			this.input.inputField.text = this.input.input.value
		}
	}
```

**Comment by CyberDex** [comment]
Fixed by #169
